avaframe/in1Data/getInput.py

```diff
@@ -172,17 +170,20 @@ def getInputDataCom1DFAPy(avaDir, cfg, flagDev=False):
     # Set directories for inputs, outputs and current work
     inputDir = os.path.join(avaDir, 'Inputs')
 
+    # Load dev flag as string
+    flagDev = cfg['flagDev']
+
     # Set flag if there is an entrainment or resistance area
     entResInfo= {'flagEnt': 'No', 'flagRes': 'No'}
 
     # Initialise release areas, default is to look for shapefiles
-    if flagDev is True:
+    if flagDev == 'True':
```

Comment:
Similar to line 183, just use cfg['flagDev'] and drop line 173 and 174

Reply:
Done